java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -30,6 +30,7 @@
 import static redis_request.RedisRequestOuterClass.RequestType.LRange;
 import static redis_request.RedisRequestOuterClass.RequestType.LRem;
 import static redis_request.RedisRequestOuterClass.RequestType.LTrim;
+import static redis_request.RedisRequestOuterClass.RequestType.Lindex;
```

Comment:
The juxtaposition of "LTrim" and "Lindex" with their different casing lol. Not something we need to address immediately though.